cni/network/invoker_cns.go

```diff
@@ -53,6 +53,7 @@ func NewCNSInvoker(podName, namespace string, cnsClient cnsclient, executionMode
 	}
 }
 
+// ryand IPv6 not currently supported
```

Comment:
not sure I understand. Which scenario is this code path covering? Don't we support ipv6 in CNS now with overlay?

Reply:
I am going to remove this since it is the CNI and relates more to what is being done on Windows. Should have been removed earlier.